src/test/ui/suggestions/extern-crate-rename.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0259]: the name `m1` is defined multiple times
+  --> $DIR/extern-crate-rename.rs:16:1
+   |
+15 | extern crate m1;
+   | ---------------- previous import of the extern crate `m1` here
+16 | extern crate m2 as m1;
+   | ^^^^^^^^^^^^^^^^^^^^^^
+   | |
+   | `m1` reimported here
```

Comment:
FWIW, I find this message confusing. I think it's the phrasing "import of the extern crate `m1`", whereas I see this statement as importing the crate `m2` -- but aliasing it to `m1`. I wonder if some small tweaks to the wording might help. Or maybe it's just fine as is, it's kind of a corner case, and if all goes to plan `extern crate` is going away anyway.

Reply:
The same wording issue [happens with `use`](https://play.rust-lang.org/?gist=0d1a6128e067ed898e473dc2413488af&version=stable).